pkg/client/unversioned/clientcmd/api/helpers.go

```diff
@@ -31,6 +31,14 @@ func init() {
 	redactedBytes = []byte(string(sDec))
 }
 
+// IsConfigEmpty returns true if the config is empty.
+func IsConfigEmpty(config *Config) bool {
```

Comment:
Oh, I see. I figured there was a reason after I wrote that. You have a test
so it's not a big deal.

On Thu, Nov 5, 2015 at 2:50 PM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/client/unversioned/clientcmd/api/helpers.go
> <https://github.com/kubernetes/kubernetes/pull/16859#discussion_r44081614>
> :
>
> > @@ -31,6 +31,14 @@ func init() {
> >  	redactedBytes = []byte(string(sDec))
> >  }
> >
> > +// IsConfigEmpty returns true if the config is empty.
> > +func IsConfigEmpty(config *Config) bool {
>
> It would have to be semantic (maps are initialized in most places) but I
> think I had an import loop. Will try. On Nov 5, 2015, at 4:59 PM, Daniel
> Smith <notifications@github.com> wrote: In
> pkg/client/unversioned/clientcmd/api/helpers.go <#16859 (comment)
> <https://github.com/kubernetes/kubernetes/pull/16859#discussion_r44075601>
> >:
> @@ -31,6 +31,14 @@ func init() { redactedBytes = []byte(string(sDec)) }
> +// IsConfigEmpty returns true if the config is empty. +func
> IsConfigEmpty(config *Config) bool {
> reflect.DeepEqual w/ Config{} seems more maintianable? — Reply to this
> email directly or view it on GitHub <
> https://github.com/kubernetes/kubernetes/pull/16859/files#r44075601>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/16859/files#r44081614>.
>